### PR TITLE
fix test set model,  use same test set and train set

### PR DESCRIPTION
**Pull request by ddapueto**
option to change to xgboost ( candidate model)

### CODE REVIEW

**Comment by johnblanco** [comment]
lo cierro porque ya esta incorporado por otro lado
